ocr/static/ocr/js/upload_form.js

```diff
@@ -1,9 +1,9 @@
 /* eslint no-unused-vars:0 */
-function submitForm(form) {
-  const url = form.attr('action');
-  const formData = {};
+function submitForm (form) {
```

Comment:
Codacy has a fix for the issue: [Unexpected space before function parentheses.](https://app.codacy.com/gh/PrynsTag/oneBarangay/pullRequest?prid=8186851)
```suggestion
function submitForm(form) {
```